ui-meta/pangeo.yml

```diff
@@ -0,0 +1,57 @@
+name: "Pangeo"
+label: "Pangeo"
+description: "Single-master Kubernetes cluster with Pangeo."
+
+parameters:
+  - name: "cluster_version"
+    label: "Version"
+    description: "The Kubernetes version to use."
+    kind: "choice"
+    options:
+      choices: ["1.12", "1.13"]
+
+  - name: "cluster_num_workers"
+    label: "Worker nodes"
+    description: "The number of worker nodes in the cluster."
+    kind: "integer"
+    options:
+      min: 3
+
+  - name: "cluster_master_flavor"
+    label: "Master size"
+    description: "The size to use for the master node."
+    kind: "cloud.size"
+    options:
+      min_cpus: 2
+      min_ram: 2048
+    immutable: true
+
+  - name: "cluster_worker_flavor"
+    label: "Worker size"
+    description: "The size to use for the worker nodes."
+    kind: "cloud.size"
+    options:
+      min_cpus: 2
+      min_ram: 2048
+    immutable: true
+
+  - name: "cluster_root_volume_size"
+    label: "Root volume size (GB)"
+    description: "The size of the root volume of cluster nodes."
+    kind: "integer"
+    options:
+      min: 10
+    immutable: true
+    default: 40
+
+  - name: "pangeo_username"
+    label: "Pangeo cluster initial user with admin privileges"
```

Comment:
This is a bit long. Probably "Admin username" is sufficient - they already know it is Pangeo.